lib/api-group.js

```diff
@@ -148,9 +148,9 @@ class ApiGroup {
    * @param {callback} cb - The callback that handles the response
    */
   patch(options, cb) {
-    this._request('PATCH', Object.assign({}, options, {
+    this._request('PATCH', Object.assign({
       headers: { 'content-type': 'application/strategic-merge-patch+json' }
-    }), cb);
+    }, options), cb);
```

Comment:
should we do a recursive merge so that headers that arent `content-type` can be specified? Not sure if thats useful but thought I'd ask.